main.js

```diff
@@ -1156,6 +1161,22 @@ function setMeta() {
     };
     tasks.push(obj);
 
+    obj = {
+        _id:       id + '.eventLoopLag',
+        type:      'state',
+        common: {
+            name:  'Controller - 15 seconds average of event nodejs loop lag in ms',
+            desc:  'Average nodejs event loop lag in ms',
```

Comment:
This description is shorter than the name, does that make sense? Otherwise:
```diff
- 'Average nodejs event loop lag in ms'
+ 'Average Node.js event loop lag in ms'
```